addons/hr/data/hr_demo.xml

```diff
@@ -99,13 +99,15 @@ Good language skills, other than English (Dutch and French preferred, others wel
           <field name="department_id" ref="dep_ps"/>
           <field name="no_of_recruitment">5</field>
           <field name="contract_type_id" ref="contract_type_interim"/>
+          <field name="description">We are currently looking for someone like that to join our Consultant team.</field>
       </record>
 
       <record id="job_developer" model="hr.job">
           <field name="name">Experienced Developer</field>
           <field name="department_id" ref="dep_rd"/>
           <field name="no_of_recruitment">5</field>
           <field name="contract_type_id" ref="contract_type_permanent"/>
+          <field name="description">We are currently looking for someone like that to join our Web team. Someone who can snap out of coding and perform analysis or meet clients to explain the technical possibilities that can meet their needs.</field>
```

Comment:
Why not keep the newlines?

Reply:
Due to the job_hrm, description and requirement --> Lorem Ipsum which is not wrapped. 

But you are right, that makes no sense, already updated ! 

Thanks